index.html

```diff
@@ -15,12 +15,72 @@
     <link rel="stylesheet" href="css/layout.css" />
     <link rel="stylesheet" href="css/styles.css" />
   </head>
+
   <body>
-    <header>
-      <div class="container"></div>
+    <header id="purple-background">
+      <div class="container">
+        <div class="row">
+          <h1 id="header-style">Code Journal</h1>
+        </div>
+      </div>
     </header>
     <main class="container">
-      <div data-view="entry-form"></div>
+      <div data-view="entry-form">
+        <form id="note-form">
+          <div class="row">
+            <div class="column-full">
+              <h2>New Entry</h2>
+            </div>
+          </div>
+          <div class="row">
+            <div class="column-half">
+              <div class="photo">
+                <label for="photo" id="img-button"></label>
+                <img
+                  id="image"
+                  src="images/placeholder-image-square.jpg"
+                  alt="Photo place holder" />
+              </div>
+            </div>
+            <div class="column-half">
+              <div class="title-for-note">
+                <label class="spacing" for="noteTitle">Title</label> <br />
```

Comment:
You shouldn't use `<br />` to force elements on a new line. You should only use it to force text within an element onto a new line. If you want the `label` to be on it's own line, think about `block` vs `inline`